webapp/packages/plugin-data-spreadsheet-new/src/DataGrid/DataGridTable.tsx

```diff
@@ -123,8 +123,8 @@ export const DataGridTable = observer<IDataPresentationProps<any, IDatabaseResul
     },
   });
 
-  const tableData = useTableData(model, resultIndex, dataGridDivRef);
-  const gridSelectionContext = useGridSelectionContext(tableData, selectionAction);
+  const [tableData, tableState] = useTableData(model, resultIndex, dataGridDivRef);
```

Comment:
please don't use react's pattern to return array of values from hooks, it's must be or paired data: value/set or get/set or just single value